scripts/start.ps1

```diff
@@ -11,6 +11,31 @@ param (
     [switch]$Run
 )
 
+# RunAs admin if not already
```

Comment:
how about doing it like following example?
I've reworked layouting a bit to match the rest of the utility and made it use ps7 if possible.

```
# Check if the current user is an administrator
if (-not ([Security.Principal.WindowsPrincipal]([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)) {
    
    # Determine which PowerShell executable to use
    if (Get-Command "pwsh" -ErrorAction SilentlyContinue) {
        $shell = "pwsh.exe"
    } else {
        $shell = "powershell.exe"
    }

    # Elevate the script and exit the current non-elevated script
    $PwshArgList = @(
        "-NoLogo",                                  # Don't print PowerShell header in CLI
        "-NoProfile",                               # Don't load PowerShell profile
        "-File", $MyInvocation.MyCommand.Source,    # Script path
        $args | ForEach-Object { $_ }               # Script arguments
    ) | ForEach-Object { "`"$_`"" }

    Start-Process -FilePath $shell -ArgumentList $PwshArgList -Verb RunAs
    exit
}
```